test/mouse/mouse.h

```diff
@@ -0,0 +1,13 @@
+TEST_CASE("[mouse] mouse movement absolute") {
```

Comment:
This could be better reworded as `[mouse] move absolute` since it's the name of the function and also more concise.